Plutarch/Builtin/ByteString.hs

```diff
@@ -218,6 +265,57 @@ phexByteStr = punsafeConstantInternal . PLC.someValue . BS.pack . f
     f [_] = error "UnevenLength"
     f (x : y : rest) = (hexDigitToWord8 x * 16 + hexDigitToWord8 y) : f rest
 
+{- | Convert a 'PByteString' into a 'PInteger', as per
+[CIP-121](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0121#builtinbytestringtointeger).
+
+@since WIP
+-}
+pbyteStringToInteger ::
+  forall (s :: S).
+  Term s PEndianness ->
+  Term s (PByteString :--> PInteger)
+pbyteStringToInteger e = plam $ \bs ->
+  punsafeBuiltin PLC.ByteStringToInteger # punsafeCoerce @PBool e # bs
```

Comment:
```suggestion
  punsafeBuiltin PLC.ByteStringToInteger # pto e # bs
```
does this work?

Reply:
just `pbyteStringToInteger = punsafeBuiltin PLC.ByteStringToInteger` is enough